docs/adoption.md

```diff
@@ -95,6 +95,13 @@ Now you have added all the resources that you need in Azure in Terraform, you ar
 
 _There will be some future work to prevent the manual entry of the AZURE STATIC WEB APPS API TOKEN secret_
 
+Once the App has been deployed, and all the API's have been deployed, navigate to the static web app, under settings, select configuration. Add the environment variables below.
+
+| Secret name              | Description                                                                                    |
+| ------------------------ | ---------------------------------------------------------------------------------------------- |
+| `REPAIRS_API_BASE_URL`   | Housing repairs online API URL, this can obtained from the App Service the API was deployed to |
+| `REPAIRS_API_IDENTIFIER` | A unique identifier used to validate access in production                                      |
```

Comment:
We don't mention where we get this identifier from. Should we add a hint for this?

Reply:
Good point. This is how we are referencing throughout the documentations so it might be a wider change as we also need to document how to generate these.